run-edep-sim/run_edep_sim.sh

```diff
@@ -39,4 +39,4 @@ run edep-sim -C -g "$ND_PRODUCTION_GEOM_EDEP" -o "$edepRootFile" -e "$nEvents" \
     <(echo "$edepCode") "$ND_PRODUCTION_EDEP_MAC"
 
 mkdir -p "$outDir/EDEPSIM/$subDir"
-mv "$edepRootFile" "$outDir/EDEPSIM/$subDir"
+mv "$edepRootFile" "$outDir/EDEPSIM/$subDir"
```

Comment:
This change doesn't seem related to this issue. If it is a bug and needs to be fixed please open a dedicated issue